loader/loader.go

```diff
@@ -671,6 +684,9 @@ func (l *Loader) Restore(ctx context.Context) error {
 	// reset some counter used to calculate progress
 	l.totalDataSize.Store(0)
 	l.finishedDataSize.Store(0) // reset before load from checkpoint
+	l.dbTableDataTotalSize = make(map[string]map[string]*atomic.Int64)
```

Comment:
prefer move the initialization to `l.prepare()`

Reply:
commet on L684: `reset some counter used to calculate progress`